lib/main_common.pm

```diff
@@ -2557,6 +2557,7 @@ sub load_extra_tests_kernel {
     loadtest "kernel/module_build";
     loadtest "kernel/tuned";
     loadtest "kernel/fwupd" if is_sle('15+');
+    loadtest "kernel/bpftrace" if is_tumbleweed;
```

Comment:
We need to run it on 15-sp5 as well.

Reply:
Thanks, I was still looking for the ticket which had the SP